### PR TITLE
Edit FARM strategies

### PR DESCRIPTION
**Pull request by chrouzz**

### CODE REVIEW

**Comment by bonustrack** [comment]
@e-asia Can you check this PR again?

**Review by e-asia — APPROVED** [review]

File: spaces/farm/index.json [hunk 1]
```diff
@@ -10,5 +10,12 @@
...
   "min": 0,
-  "invalid": []
+  "invalid": [],
+  "strategies": [
+    ["erc20BalanceOf", {
+      "address": "0x59258F4e15A5fC74A7284055A8094F58108dbD4f",
+      "symbol": "FARM",
+      "decimals": 18
+    }]
+  ]
```
Comment (e-asia) [inline spaces/farm/index.json:20]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/farm/index.json (hunk 1 above)
Comment (chrouzz) [reply spaces/farm/index.json:22]: I think I already have a chainId and decimals in the index.json

Comment (chrouzz) [reply spaces/farm/index.json:22]: you mean the chainId and decimals is for the voting strategie?

**Review by e-asia — APPROVED** [review]

**Comment by e-asia** [comment]
@chrouzz We merged your PR but had to update the strategy here #114. All the old proposals will have the result of the new strategy. That means a different result compared to the previous strategy.

**Pull request merged**